src/mame/drivers/y2.cpp

```diff
@@ -125,7 +125,7 @@ ROM_END
    There is a PS2 version with the Unlimited Match title screen, but unless it's used for a different region the arcade doesn't show that title, only announces it. */
 GAME( 2009, kof2002um,  0,         system_board_y2, system_board_y2,  system_board_y2_state, empty_init, ROT0, "SNK Playmore / New Channel", "The King of Fighters - Fuchou Zhi Lu/Road to Revenge / The King of Fighters 2002 Unlimited Match (China)", MACHINE_IS_SKELETON ) // also Export?
 GAME( 2009, kof2002umj, kof2002um, system_board_y2, system_board_y2,  system_board_y2_state, empty_init, ROT0, "SNK Playmore",               "The King of Fighters 2002 Unlimited Match (Japan)",                                                        MACHINE_IS_SKELETON )
-GAME( 2009, higurashi,  0,         system_board_y2, system_board_y2,  system_board_y2_state, empty_init, ROT0, "AQ Interactive",             "Higurashi no Naku Koro NiJong (Japan)",                                                                    MACHINE_IS_SKELETON )
+GAME( 2009, higurashi,  0,         system_board_y2, system_board_y2,  system_board_y2_state, empty_init, ROT0, "AQ Interactive",             "Higurashi no Naku Koro Ni Jong (Japan)",                                                                   MACHINE_IS_SKELETON )
```

Comment:
Can we lowercase the "Ni" on these typos too? It's grammatically a particle, like the lowercase "no" in the title.

Reply:
Thanks for the advice.  I’ll get to it as soon as I can.